src/common/types/store-data/path-snippet-data.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export interface PathSnippetData {
```

Comment:
It looks like the convention for this data types is to use the suffix `StoreData` so this should be renamed to `PathSnippetStoreData`

Reply:
Got it, renaming!